apis/meta/v1alpha1/log_level.go

```diff
@@ -164,3 +164,36 @@ func (l HarborLogLevel) Notary() NotaryLogLevel {
 		return NotaryFatal
 	}
 }
+
+// +kubebuilder:validation:Type=string
+// +kubebuilder:validation:Enum={"debug","info","warning","error","fatal","panic"}
+// +kubebuilder:default="info"
+// TrivyLogLevel is the log level for Trivy.
+type TrivyLogLevel string
+
+const (
+	TrivyDebug        TrivyLogLevel = "debug"
+	TrivyInfo         TrivyLogLevel = "info"
+	TrivyWarning      TrivyLogLevel = "warning"
+	TrivyError        TrivyLogLevel = "error"
+	TrivyFatal        TrivyLogLevel = "fatal"
+	TrivyPanic        TrivyLogLevel = "panic"
+	TrivyDefaultLevel TrivyLogLevel = TrivyInfo
+)
+
+func (l HarborLogLevel) Trivy() TrivyLogLevel {
```

Comment:
The purpose of this function is to make sure the log level settings are kept consistent with harbor log level settings, right?

Reply:
You are right. The goal is to get Trivy log level from Harbor one.